mindsdb/integrations/handlers/together-ai-handler/requirements.txt

```diff
@@ -0,0 +1 @@
+together
```

Comment:
I'd suggest pinning the version so that the handler doesn't randomly break if there is a major version release down the line.